### ISSUE TITLE
QC the rs-fMRI

### ISSUE THREAD

**Issue by lunebellec** [ccna-biomarkers/biomarkers-ccna-ccnv#11]

**Comment by ltetrel** [comment]
QC tool: https://simexp-documentation.readthedocs.io/en/latest/giga_preprocessing/qc.html

**Comment by lunebellec** [comment]
Will use the fmriprep QC, and the wrapper created by Désirée. 

Also need to agree about some rough QC directions. 

@ltetrel can help with producing statistics of QC metrics.

**Comment by ltetrel** [comment]
Group-level summary maps of coregistration procedures: https://iopscience.iop.org/article/10.1088/1742-6596/341/1/012032/pdf#page=13
FD score per subject: https://docs.google.com/presentation/d/15sA8vUcIZXwG33hd0txlD-htIJI4XVfJyYD2N0Kq5JA/edit#slide=id.gd859b62f46_0_0

**Comment by htwangtw** [comment]
For QC metric, the NKI dataset has released their [QC guildline](http://fcon_1000.projects.nitrc.org/indi/enhanced/qc.html) for resting state data. Might be helpful

**Comment by ltetrel** [comment]
I also pinged @bpinsard and @vborghe for their code. She mentionned https://github.com/RainCloudPlots/RainCloudPlots waiting if they have also the scripts.

**Comment by lunebellec** [comment]
@htwangtw this is definitely helpful. However, there are no examples provided for the different artefacts. I guess this could be fleshed out a little. We may also want to use the QC Yassine developed for structural registration which does have a clean documentation and documented reliability https://pubmed.ncbi.nlm.nih.gov/32180712/ but it's not suitable for, say, freesurfer, and only covers a small part of the protocol.

**Comment by htwangtw** [comment]
> @htwangtw this is definitely helpful. However, there are no examples provided for the different artefacts. I guess this could be fleshed out a little. We may also want to use the QC Yassine developed for structural registration which does have a clean documentation and documented reliability https://pubmed.ncbi.nlm.nih.gov/32180712/ but it's not suitable for, say, freesurfer, and only covers a small part of the protocol.

That's a really good point - I definitely overlooked the lack of examples as I am familiar what each of those instructions means. The figure from Yassine's paper is really good. 
For Freesurfer - I have very limited experience of doing cortical thickness correction. From what I understand the level of QC depends on the level of information we need from it. Again this is another thing I learned through sitting down with experts. Can look around for some thing more comprehensive.

**Comment by lunebellec** [comment]
for freesurfer @dllussier has some experience QCing the surface. 

re familiarity with the artefacts, my experience trying to quantify and standardize QC is that it's hard to reach a consistent decision, even for seemingly easy checks. Unfortunately I am not aware of a quantitative evaluation of a QC protocol with that level of detail. 

Also note that Yassine's protocol includes a training session, which I think is critical to properly train raters https://www.zooniverse.org/projects/simexp/brain-match/classify

**Comment by ltetrel** [comment]
Basilae shared this repo for QC metrics analysis, I will take a look at it to adapt to ccna: https://github.com/courtois-neuromod/cneuromod_qc

**Comment by dllussier** [comment]
Hi @ltetrel it looks like that repo is still a work in progress. I am concerned about the amount of time needed to get it to a working state, adapted, and debugged. For the resting state, I think it is still a good idea to do a standard visual inspection of the data as well as supply some standard QC metrics (motion, etc).  The QC is starting next week and will not take very long to complete given the sample size.

**Comment by ltetrel** [comment]
Indeed there are lot empty files there, I will check with him.
Also the goal here is to not make redundant work on the QC metrics visualization (FD, or mean mask volume), this is not directly related to the visual inspection itself (that needs to be performed on each scan anyway, and not related to this repo).
Basically the Neuromod team has the scripts for some metrics we want, we should re-use that.

**Comment by dllussier** [comment]
My question then is, what resting-state metrics will their scripts provide that fMRIprep and MRIqc will not be able to?

**Comment by htwangtw** [comment]
fMRIprep definitely don't have any utility to pull the individual stats into summary graphs. I believe @ltetrel  is looking for the script for the summary figures.

**Comment by ltetrel** [comment]
So fMRIprep provide [individual reports](https://fmriprep.org/en/stable/outputs.html#confounds-and-carpet-plot-on-the-visual-reports) (this is what is showed to the user by https://simexp-documentation.readthedocs.io/en/latest/giga_preprocessing/qc.html) but indeed I thought it was clear that we are talking about summary figures here as @htwangtw just pointed out.

**Comment by amanbadhwar** [comment]
My two cents is we proceed with the visual QC. We can provide the QC metrics from fMRIprep and MRIqc (like fd, snr, etc) in addition to the visual. This project has been on hold long enough, and needs to be wrapped up ASAP.

**Comment by dllussier** [comment]
I agree with @amanbadhwar. This has already taken a long time to get moving and should not wait longer. Visualizations summarizing QC metrics can be a discussion for after or in a separate issue please. We really need to focus on getting the QC finished so that we can proceed with the rest.
